sbi/inference/snpe/snpe_base.py

```diff
@@ -269,7 +263,7 @@ def _run_simulations(
 
             x = self._batched_simulator(theta)
 
-        return theta, x, self._mask_sims_from_prior(round_, theta.size(0))
+        return theta, x
```

Comment:
If the method `_mask_sims_from_prior` (below) still needed at all?

Reply:
Unfortunately yes, for still need it (for the `combined_loss` in snpe)